game-service/src/main/java/org/libertybikes/game/round/service/GameRoundService.java

```diff
@@ -50,6 +51,14 @@ public GameRound getRound(@PathParam("roundId") String roundId) {
         return allRounds.get(roundId);
     }
 
+    @GET
+    @Path("/roundStatus/{roundId}")
+    @Produces(MediaType.APPLICATION_JSON)
+    public JsonObject roundStatus(@PathParam("roundId") String roundId) {
```

Comment:
This method is still duplicated with the existing `getRound(String roundId)` endpoint. Even though it returns type `GameRound` instead of `JsonObject`, the returned GameRound will automatically be converted to JSON because the getRound method has `@Produces(MediaType.APPLICATION_JSON)`